### DISCUSSION TITLE
基于UP主的 程序 改了一个摄像头应用 ：）

### DISCUSSION THREAD

**Discussion by wenFrankieeee** [peng-zhihui/ElectronBot#148] [General]
如视

https://user-images.githubusercontent.com/101490638/162747310-68342c04-f03a-4a7f-96ac-6196b951247f.mp4

频

**Comment by coolflyreg** [comment]
没太看明白咋回事。你是开发了手机版驱动吗？

**Comment by peng-zhihui** [comment]
WiFi图传吧哈哈

**Reply by wenFrankieeee** [reply]
哈哈，还真不是，简单把你sample改了，video capture(0)。

**Comment by wenFrankieeee** [comment]
https://user-images.githubusercontent.com/101490638/162791191-62c49a18-6900-48c2-80ba-a0f5cdbfd39c.mp4
就是这样
